websubhub-ballerina/tests/basic_hub_test.bal

```diff
@@ -23,7 +23,7 @@ boolean isValidationFailed = false;
 
 http:Client httpClient = checkpanic new("http://localhost:9090/websubhub");
 
-listener Listener functionWithArgumentsListener = new(9090);
+listener Listener functionWithArgumentsListener = new(9090, "https://sample.hub.com");
```

Comment:
[URGENT] Do we have to ask this explicitly? Wouldnt we be able to get this from the underlying http client?